lib/screenplay/config/config.ex

```diff
@@ -87,6 +92,25 @@ defmodule Screenplay.Config.PermanentConfig do
     end
   end
 
+  defp check_for_duplicate_screen_ids(deserialized, places_and_screens) do
+    %PendingConfig{screens: existing_screens} = PendingConfig.from_json(deserialized)
+
+    all_screen_ids =
```

Comment:
We should include the live screens in this list as well. There's technically no issue with a pending and live screen sharing an ID at this point, but it will be an issue when the pending screen is published.